bls12_381/Forc.toml

```diff
@@ -5,4 +5,4 @@ license = "Apache-2.0"
 name = "bls12_381"
 
 [dependencies]
-utils = { path = "../utils" }
\ No newline at end of file
+utils = { path = "../utils" }#todo fix
```

Comment:
this is for future reference when we continue with those old projects (bls12-381 and ed25519). The "old" utils were out of date so we renamed them to `utils_old` and added an updated version.